### DISCUSSION TITLE
Meshtastic Firmware 2.5.14.f2ee0df Alpha Web Server

### DISCUSSION THREAD

**Discussion by KG5RDF** [meshtastic/Discussions#94] [Firmware]
The Web Server under journal is loaded without errors. Its there, but blank. This is on a raspberry pi 3b+. The older Beta version has no issue. This is a firmware issue, not a network issue.

**Comment by fifieldt** [comment]
Please refer to: https://github.com/meshtastic/firmware/discussions/5381

**Comment by KG5RDF** [comment]
Thanks!!
-------- Original message --------From: Tom Fifield ***@***.***> Date: 11/26/24  3:56 AM  (GMT-06:00) To: meshtastic/Discussions ***@***.***> Cc: KG5RDF ***@***.***>, Author ***@***.***> Subject: Re: [meshtastic/Discussions] Meshtastic Firmware 2.5.14.f2ee0df Alpha Web Server (Discussion #94) 
Please refer to: meshtastic/firmware#5381
